app/components/embed/companion_windows_component.html.erb

```diff
@@ -37,20 +37,13 @@
           <dl data-controller="iiif-metadata" class="metadataList left-drawer-content" data-action="iiif-manifest-received@window->iiif-metadata#drawMetadata"></dl>
         </section>
 
-        <section id="media-content" role="tabpanel" hidden>
-          <div class="header-background">
-            <h3>Media content</h3>
-          </div>
-          <ul class="left-drawer-content" data-controller="content-list" data-action="thumbnails-found@window->content-list#draw" role="tablist"></ul>
+        <section id="object-content" role="tabpanel" hidden>
+          <%= render ContentListComponent.new(viewer:, heading: viewer.content_heading) %>
```

Comment:
How will this affect the media component?

Reply:
Media content is unaffected partly because of this (i.e. the component does nothing if not in the PDF viewer and a PDF file in the object): https://github.com/sul-dlss/sul-embed/pull/2082/files#diff-42a38cc64b6677f81d752edaa0880848b977216a28209c764ffeaf15ff57699fR30-R35

But also, the javascript method that draws the media thumbnails completely replaces the HTML content with the dynamically generated list of thumbnails anyway: https://github.com/sul-dlss/sul-embed/pull/2082/files#diff-72decc26473133ce4c39fa2e64877e55a38bbb13fb955b948cb6efd805155521L7-L9

So the component not rendering anything for media just saves us some time producing something that would be get replaced later anyway.

Screen shot: 
<img width="662" alt="Screen Shot 2024-01-11 at 9 12 49 AM" src="https://github.com/sul-dlss/sul-embed/assets/47137/5e395c68-f26e-441f-92d2-6072a53cc9de">

Reply:
Why don't we render this data out of the manifest.json? It seems like we're moving toward using the manifest for more of the information, so why not here too?

Reply:
I did consider doing it client side in javascript parsing the manifest like how it's done for the media thumbnails, but it was more complicated and required writing more code, which seemed unnecessary given the simplicity of what needs to happen for PDFs.  So basically because it's easier to do server side with existing methods (see https://github.com/sul-dlss/sul-embed/blob/bb92d7125aa7c74d5f870eb637f9a8756e2429e4/app/models/embed/purl.rb#L58-L60) rather than client side in javascript, given that we can re-use existing methods and technique (e.g. already used by the download panel in the same player).